lib/request/request.js

```diff
@@ -22,6 +21,22 @@ function httpRequest (origRequestOptions) {
 
   debug('REQUEST:', requestOptions)
 
+  /*!
+  * isArrayBuffer v1.0.0
+  * https://github.com/fengyuanchen/is-array-buffer
+  *
+  * Copyright (c) 2015-2017 Chen Fengyuan
+  * Released under the MIT license
+  *
+  * Date: 2017-07-26T11:00:44.931Z
+  */
+  function isArrayBuffer (value) {
+    var hasArrayBuffer = typeof ArrayBuffer === 'function'
+    var toString = Object.prototype.toString
+    return hasArrayBuffer && (value instanceof ArrayBuffer || toString.call(value) === '[object ArrayBuffer]')
+  }
+  // --- end isArrayBuffer
```

Comment:
could you please move it into ` lib/request/is-array-buffer.js`, mention that it’s a temporary workaround and reference https://github.com/octokit/rest.js/issues/774